Makefile

```diff
@@ -23,7 +23,9 @@ js/test: js/vimlparser.js
 	test/run_command.sh node js/vimlparser.js
 
 py/test: py/vimlparser.py
-	test/run_command.sh python py/vimlparser.py
+	for python in $(VIMLPARSER_PYTHON_LIST); do \
+		test/run_command.sh $$python py/vimlparser.py; \
+	done
```

Comment:
Hmm, this only checks for syntax errors I assume?

We would get this also when using flake8 (i.e. the "checkpy" target, #122) - not sure if it would need to run flake8 both via py2 and py3 (usually using py3 only is fine).

Reply:
This assumption was wrong - it actually runs tests, and it makes sense to have this then with py2 and py3 I guess.